frontend_tests/node_tests/user_topics.js

```diff
@@ -85,14 +85,14 @@ test("get_mutes", () => {
     assert.deepEqual(all_muted_topics, [
         {
             date_muted: 1577836700000,
-            date_muted_str: "Dec\u00A031,\u00A02019",
+            date_muted_str: "Dec 31, 2019",
             stream: devel.name,
             stream_id: devel.stream_id,
             topic: "java",
         },
         {
             date_muted: 1577836800000,
-            date_muted_str: "Jan\u00A001,\u00A02020",
+            date_muted_str: "Jan 01, 2020",
```

Comment:
@shameondev am I correct in reading this as a functional change -- we're no longer using non-breaking space characters in `date_muted_str`? Can you check if this results in ugly line-wrapping in the "muted topics" panel of user settings for mobile sizes?

If so, I could imagine us doing post-processing to replace spaces with non-breaking spaces for this string.

Reply:
Ahh, just read on https://github.com/zulip/zulip/pull/24015/files#r1065752086; can you check the CSS for this column in the `Muted topics` panel? I think that's the only place this particular string format is used.

Reply:
I may have misunderstood your concerns about the "date muted" column in the muted topics settings. From my perspective, everything looks good here.
![localhost_9991_(iPhone SE)](https://user-images.githubusercontent.com/53193850/213884607-2ee5231d-2819-40e5-88b6-2722a05f70b3.png)

Reply:
Well, I imagine if you have a topic that's like 60 characters long, one might have a wrapping issue.

Reply:
Yeah, it looks good to me as well. Speaking of header word wrapping i think that this is an intentional design choice, as the` .wrapped-table` class has the `word-break` property set to `break-word`. I don't like the visual part, but i am certain there are some reasons for this.

<img width="354" alt="image" src="https://user-images.githubusercontent.com/53193850/214242587-2dd8e449-9cab-47dd-a79e-30954178141b.png">